shared/components/CommunityAbout/index.scss

```diff
@@ -21,16 +15,19 @@
       margin-bottom: 0px;
     }
 
-    @media (min-width: 768px) {
+    @media (min-width: $tablet-portrait-width) {
```

Comment:
Good point!

.EventDetails, .NextCommunityEvent, .NextConferenceEvent and .TicketStatus refers to 768px

I will go throw and replace these.


.SiteFooter has 750px (but @lpil did tell me there was a particular reason for that)